src/commands/walkthroughCommands.ts

```diff
@@ -113,6 +119,57 @@ export class WalkthroughCommands {
   }
 
   async installDbt(): Promise<void> {
+    if (
+      workspace
+        .getConfiguration("dbt")
+        .get<string>("dbtIntegration", "core") === "cloud"
+    ) {
+      this.installDbtCloud();
+    } else {
+      this.installDbtCore();
+    }
+  }
+
+  private async installDbtCloud(): Promise<void> {
+    let error = undefined;
+    await window.withProgress(
+      {
+        title: `Installing dbt cloud...`,
+        location: ProgressLocation.Notification,
+        cancellable: false,
+      },
+      async () => {
+        try {
+          const { stderr } = await this.commandProcessExecutionFactory
+            .createCommandProcessExecution({
+              command: this.pythonEnvironment.pythonPath,
+              args: ["-m", "pip", "install", `dbt`],
```

Comment:
dbt suggests adding "--no-cache-dir", see https://docs.getdbt.com/docs/cloud/cloud-cli-installation?install=pip#install-dbt-cloud-cli-in-pip. Also maybe use normal quotes for dbt.